src/frontends/paddle/src/op/set_value.cpp

```diff
@@ -0,0 +1,156 @@
+    starts_node = std::make_shared<default_opset::ScatterNDUpdate>(starts_node, axes_node, starts);
+
+    // 3.2 get ends node
+    ends_node =
+        default_opset::Constant::create(element::i64, {static_cast<size_t>(dims)}, std::vector<int64_t>(dims, -1));
+    ends_node = std::make_shared<default_opset::ScatterNDUpdate>(ends_node, axes_node, ends);
+
+    // 3.3 get steps node
+    steps_node =
+        default_opset::Constant::create(element::i64, {static_cast<size_t>(dims)}, std::vector<int64_t>(dims, 1));
+    steps_node = std::make_shared<default_opset::ScatterNDUpdate>(steps_node, axes_node, steps);
+
+    // 4.get target value shape
+    // 4.1 end - start
+    Output<Node> value_shape_update_node = std::make_shared<default_opset::Subtract>(ends, starts);
+    // 4.2 ( end - start ) / step
+    value_shape_update_node = std::make_shared<default_opset::Divide>(value_shape_update_node, steps);
+    // 4.3 ceil(( end - start ) / step)
+    value_shape_update_node = std::make_shared<default_opset::Ceiling>(value_shape_update_node);
+    // 4.4 update
+    const auto value_target_shape =
+        std::make_shared<default_opset::ScatterNDUpdate>(input_shape, axes_node, value_shape_update_node);
+
+    // 4.5 broadcast
+    value_node = std::make_shared<default_opset::Broadcast>(value_node, value_target_shape);
+
+    // get total number of elements
+    const auto numel_node = std::make_shared<default_opset::ReduceProd>(input_shape, zero_node);
+    // generate indices from 0 to numel - 1
+    Output<Node> range_node = std::make_shared<default_opset::Range>(zero_node, numel_node, one_node, element::i64);
+    // reshape to input_shape
+    range_node = std::make_shared<default_opset::Reshape>(range_node, input_shape, true);
+    // slice range node, get the indices thta to be updated
+    Output<Node> sliced_range_node = std::make_shared<default_opset::StridedSlice>(range_node,
+                                                                                   starts_node,
+                                                                                   ends_node,
+                                                                                   steps_node,
+                                                                                   std::vector<int64_t>(dims),
+                                                                                   std::vector<int64_t>(dims));
+
+    // flatten input, upadte_value and sliced_range_node
+    input_node = std::make_shared<default_opset::Reshape>(input_node, reshape_flatten, true);
+    sliced_range_node = std::make_shared<default_opset::Reshape>(sliced_range_node, reshape_flatten, true);
+    value_node = std::make_shared<default_opset::Reshape>(value_node, reshape_flatten, true);
+
+    // update value to input according to sliced_range_node
+    input_node = std::make_shared<default_opset::ScatterUpdate>(input_node, sliced_range_node, value_node, zero_node);
```

Comment:
From the error log of ScatterUpdate, the shape of input/slice/value is incorrect. Recommend to use ScatterNDUpdate to update value

Reply:
The shape error is due to the slice operation, which will return the maximum number in int32
for example:
```python
x[::2] = 1
```
so start, end and step are 0, 214748364, 2

Reply:
It's weird that if start or end are minus, it will throw `StridedSlice node doesn't have compiled executor!`. But I have converted them to positive in fucntion `handle_minus_index`.

Reply:
Please check the spec of https://docs.openvino.ai/latest/openvino_docs_ops_movement_ScatterUpdate_3.html . Need to reshape the indice and value if want to use the ScatterUpdate. Use ScatterNDUpdate may be easier here.
About the StridedSlice error, it is another issue. May need to debug

Reply:
done, but only the cases whose all params(starts, ends) are positive can pass the tests